### DISCUSSION TITLE
Sitecore integration with Vercel

### DISCUSSION THREAD

**Discussion by cresencio** [vercel/next.js#56186] [Help]
### Summary

Just signed up for Sitecore's [front-end developer trial](https://www.sitecore.com/knowledge-center/getting-started/developer-trial) and I was wondering if setting that up would allow me to use the same API keys for [Vercel's Sitecore template](https://vercel.com/templates/next.js/sitecore-starter). If not, is there another route I could take that would allow me to play around with a Sitecore instance with Vercel integration? 

Thank you!

**Comment by jst-cyr** [comment]
Hey there! Jason from Sitecore's DevRel team... We do have free trials for a few of the Sitecore products including Send and OrderCloud, and it seems you've already found our developer sandbox trial that uses our traditional platform and Sitecore JSS to get you an idea of some of the capabilities for developing. 

Unfortunately, that trial you signed up for is not for XM Cloud and doesn't use our Edge GraphQL API, so it does not get you the same API keys that you would need for that Vercel template. We don't currently offer a trial sandbox for XM Cloud but it is definitely something we hear a lot of demand for and are always evaluating adding that in the future!
